pkg/dbutil/index.go

```diff
@@ -113,6 +113,71 @@ func FindSuitableIndex(ctx context.Context, db *sql.DB, schemaName string, table
 	return c, nil
 }
 
+// FindAllIndex returns all index, order is pk, uk, and normal index.
+func FindAllIndex(tableInfo *model.TableInfo) []*model.IndexInfo {
+	var primaryIndex *model.IndexInfo
+	uniqueIndices := make([]*model.IndexInfo, 0, 1)
+	normalIndices := make([]*model.IndexInfo, 0, 1)
+
+	for i, index := range tableInfo.Indices {
+		if index.Primary {
+			primaryIndex = tableInfo.Indices[i]
```

Comment:
```suggestion
			primaryIndex = index
```

(similar for the two `tableInfo.Indices[i]` below, if you decide to keep this implementation)

Reply:
index is a pointer, if use index, at the last, primaryIndex, uniqueIndex will use same pointer.
it dosen't matter, I have already removed these code.